core-bundle/src/Routing/Page/CompositionAwareInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Routing\Page;
+
+use Contao\PageModel;
+
+interface CompositionAwareInterface
+{
+    /**
+     * If the page supports content composition, it's layout is defined by a Contao
```

Comment:
```suggestion
     * If the page supports content composition, its layout is defined by a Contao
```

Reply:
Fixed in de480f2f5314037da6f0ecb203fc77d093e6a205